### PR TITLE
Fix bug found by gcc 5.3

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: tests/matrix.cpp [hunk 1]
```diff
@@ -76,6 +76,17 @@ BOOST_AUTO_TEST_CASE( lookat )
     BOOST_CHECK_EQUAL( up, newUp );
 }
 
+BOOST_AUTO_TEST_CASE( nonInvertible )
+{
+    vmml::Matrix2f matrix;
+    matrix( 0, 0 ) = 0.f;
+    matrix( 1, 1 ) = 0.f;
+    matrix.inverse();
+    // A NaN never compares equal to itself.
```
Comment (rdumusc) [inline tests/matrix.cpp]: This test is testing nothing? use http://en.cppreference.com/w/cpp/numeric/math/isnan ?

Comment (eile) [inline tests/matrix.cpp]: It executed the code path. I did not find isnan, thanks, done.

**Comment by rdumusc** [comment]
+1

**Comment by eile** [comment]
Retest this please.

**Pull request merged**